homeassistant/helpers/service.py

```diff
@@ -221,6 +221,11 @@ def async_prepare_call_from_config(
                         registry, entity_ids
                     )
                 target[CONF_ENTITY_ID] = entity_ids
+                variables = dict(variables) if variables else {}
+                variables["entities"] = config.get(
```

Comment:
Is there a precedent for this variable name? We normally call it `entity_id`, eg in service schemas, also when it holds a list of entity_ids.

Reply:
No precedent. I can change it to `entity_id` if that's the convention. I guess it just seems confusing to access the entity_id by `entity_id.entity_id`...

Reply:
Yeah, I'm not sure either. Would be good with some more opinions.

Reply:
I'm going to shelve this for now. I don't think it makes sense to do it this way as it doesn't fully resolve the issue. Also no point in rushing this before beta.